Dockerfile

```diff
@@ -29,6 +34,8 @@ ARG BIN_SUFFIX
 LABEL maintainers="Intel"
 LABEL description="PMEM CSI Driver"
 
+COPY --from=build python3_100.0_all.deb /var/cache/python3_100.0_all.deb
```

Comment:
This creates a layer, which cannot be avoided easily (https://forums.docker.com/t/copy-a-file-that-doesnt-need-to-be-in-the-final-image/12627/5).

But it's small, so this shouldn't matter.

Reply:
2KB

Reply:
I think dummy package approach here is cleaner than that other hack in #783 which requires brutal altering of dpkg state.

Reply:
So shall we merge this? I think it's worthwhile.

Reply:
yes, I will merge, let's see how the devel branch images will look.